lib/convection/model/template/resource/aws_route53_recordset.rb

```diff
@@ -9,20 +9,34 @@ class Resource
         ##
         class Route53RecordSet < Resource
           type 'AWS::Route53::RecordSet', :route53_recordset
-          property :alias_target, 'AliasTarget'
+          property :alias_tgt, 'AliasTarget'
```

Comment:
> but I could also make it backwards-comparable.

Since we're bumping the version by a major version I don't mind breaking this so much. We just shouldn't release `1.0.0` until we've cleared up all these discrepancies.

We can search for our internal convection usage to see if either of these properties were used and update to use your format before releasing a new pre-release version. Can you go ahead and create a ticket in our internal issue tracker for that (CC @bcall-r7 and I)?